include/veriblock/pop/blockchain/pop/pop_state.hpp

```diff
@@ -32,27 +32,22 @@ struct PopState {
     return _endorsedBy;
   }
 
-  void insertContainingEndorsement(std::shared_ptr<endorsement_t> e) {
-    VBK_ASSERT_MSG(e != nullptr, "Inserted endorsement should not be nullptr");
-    _containingEndorsements.emplace(e->id, std::move(e));
-    setDirty();
-  }
-
   void insertEndorsedBy(const endorsement_t* e) {
     VBK_ASSERT_MSG(e != nullptr, "Inserted endorsement should not be nullptr");
     _endorsedBy.push_back(e);
-    setDirty();
```

Comment:
why?

Reply:
It is not ready for review yet. Wait a bit.